lib/util/miq_apache/miq_apache.rb

```diff
@@ -125,7 +125,7 @@ def self.status(full = true)
     end
 
     def self.version
-       MiqUtil.runcmd("rpm -qa --queryformat '%{VERSION}' httpd")
+      MiqUtil.runcmd("rpm -qa --queryformat '%{VERSION}' #{MiqApache.package_name}")
```

Comment:
This could be ```LinuxAdmin::Rpm.info(MiqApache.package_name)["version"]```

```ruby
# irb
irb(main):001:0> require 'linux_admin'
=> true
irb(main):002:0> LinuxAdmin::Rpm.info("httpd")
=> {"name"=>"httpd", "relocations"=>"(not relocatable)", "version"=>"2.2.15", "vendor"=>"Red Hat, Inc.", "release"=>"39.el6", "build_date"=>"Fri 15 Aug 2014 03", "install_date"=>"Thu 08 Jan 2015 01", "build_host"=>"x86-027.build.eng.bos.redhat.com", "group"=>"System Environment/Daemons", "source_rpm"=>"httpd-2.2.15-39.el6.src.rpm", "size"=>"3084425", "license"=>"ASL 2.0", "signature"=>"RSA/8, Fri 12 Sep 2014 08", "packager"=>"Red Hat, Inc. <http", "url"=>"http", "summary"=>"Apache HTTP Server", "description"=>"The Apache HTTP Server is a powerful, efficient, and extensible web server. "}
irb(main):003:0> LinuxAdmin::Rpm.info("httpd")["version"]
=> "2.2.15"
```